### PR TITLE
feat(database/instance): add missing tab

### PR DESCRIPTION
**Pull request by stonebuzz**
Add missing tab ```Infocom``` and ```Appliance``` for ```Database``` object.

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #number

### CODE REVIEW

**Review by orthagh — APPROVED** [review]
still draft ?

**Comment by stonebuzz** [comment]
> still draft ?

it should be good =)

File: inc/define.php [hunk 1]
```diff
@@ -649,4 +649,5 @@
     SoftwareLicense::getType()    => "pics/impact/softwarelicense.png",
     Supplier::getType()           => "pics/impact/supplier.png",
     User::getType()               => "pics/impact/user.png",
+    Database::getType()           => "pics/impact/databaseinstance.png",
```
Comment (cedric-anne) [inline inc/define.php:652]: Maybe png files should be renamed to match their corresponding itemtype.
- `databaseinstance.png` -> `database.png`
- `database.png` -> `databaseinstance.png`

Comment (stonebuzz) [reply inc/define.php]: done

**Review by trasher — APPROVED** [review]

**Pull request merged**